Makefile

```diff
@@ -512,7 +512,7 @@ COMMON_SRC = \
             fc/config.c \
             fc/fc_tasks.c \
             fc/fc_msp.c \
-            fc/mw.c \
+            fc/fc_main.c \
```

Comment:
I don't really like the name `fc_main.c` since there is nothing "main" about this file at all. It's just a collection of bits that we haven't really found a home for.

On the other hand, I can't really think of a better name.